lib/matplotlib/axes/_base.py

```diff
@@ -2810,6 +2810,19 @@ def get_xlim(self):
         """
         return tuple(self.viewLim.intervalx)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        Raise ValueError if specified axis limits are infinite.
+
```

Comment:
I'll change the name to `_validate_converted_limits` . Should I change the docstring to """ Raise ValueError if converted limits are non-finite.
Note that this functions also accepts None as a limit argument.
Returns : limit value after call to convert_xunits() or convert_yunits() or convert_zunits()""" ?